packages/tupaia-web/src/layout/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { TRANSPARENT_BLACK } from '../../constants';
+import { ExpandButton } from './ExpandButton';
+
+const MAX_SIDEBAR_EXPANDED_WIDTH = 1000;
+const MAX_SIDEBAR_COLLAPSED_WIDTH = 350;
+
+const Panel = styled.div<{
+  $isExpanded: boolean;
+}>`
+  display: flex;
+  align-items: stretch;
+  flex-direction: column;
+  align-content: stretch;
+  position: relative;
+  overflow: visible;
+  background-color: ${TRANSPARENT_BLACK};
+  pointer-events: auto;
+  height: 100%;
+  cursor: auto;
+  transition: width 0.5s ease;
+  width: ${({ $isExpanded }) => ($isExpanded ? 45 : 30)}%;
```

Comment:
```suggestion
  width: ${({ $isExpanded }) => ($isExpanded ? 45 : 30)}%;
  min-width: 335px;
```

Also let's add a min-width so it matches the behaviour of the old site.